Rewrite of Using Radicle section to capture feedback from onboarding (#49)
radicle-dev/radicle-docs: docs/using-radicle/creating-projects.md

> ---

Reviewer: The flow in this file is not that straightforward. I'd assume a more hands-on guide on how to go through the project creation flow, showing pictures of the modal in the UI explaining what the differences between the two ways are creating a project. Currently the first part tries to explain all the concepts, which might be overwhelming at first and the can be introduced more gently towards the end, when the user lands on the project page aftyer creating their first project.

id: creating-projects
title: Creating projects
---

In Radicle, local Git repositories are published and shared as [projects][pr].

> Projects contain all files of a repository and stores each file's revision
> history. Projects also have associated metadata such as a project name and
> description. 

Reviewer: ```suggestion
Projects contain all files of a repository and their revision history.
Projects also have associated metadata such as a name and description. 
```

---
id: creating-projects
title: Creating projects
---

Creating a project is as easy as initializing a repository in Git. In the
Upstream client, you can create a project from scratch or from an existing
repository. 

## Creating a new project

If starting a new project from scratch, choose `Create from a new repository` in
the **Start a new project** modal. This will prompt you to choose a location on
your machine for the repository to be created. At the moment, [projects can only
be created through the Upstream client][fa].

[Insert picture of Start a new project modal]

This will initalize a Git repository (with the name you've given it) in the
location you choose and publish it to the Radicle network. Note, that when a
project is published to the network, it will be able to be replicated by any
connected peers. 

## Publishing an existing project

If you have a repository that you'd like to publish to the Radicle network,
choose the 'Create from an exisiting repository' in the **Start a new project**
modal. This will allow you to choose the repository and branch you wish to
publish to the network.

> The name of the project on the network will be fetched from the name of the

Author: ```suggestion
The name of the project on the network will be taken from the name of the
```

---
id: creating-projects
title: Creating projects
---

Creating a project is as easy as initializing a repository in Git. In the
Upstream client, you can create a project from scratch or from an existing
repository. Creating a project on Radicle will publish it to the network, making
it available to any of your connected peers.

## Starting a new project

If starting a new project from scratch, choose `Create from a new repository` in
the **Start a new project** modal. This will prompt you to choose a location on
your machine for the repository to be created. At the moment, [projects can only
be created through the Upstream client][fa].

![Start New Project][np]

This will initalize a Git repository (with the name you've given it) in the
location you choose and publish it to the Radicle network. Note, that when a
project is published to the network, it will be able to be replicated by any
connected peers. 

## Publishing an existing project

If you have a repository that you'd like to publish to the Radicle network,
choose the 'Create from an exisiting repository' in the **Start a new project**

> modal. This will allow you to choose the repository and branch you wish to
> publish to the network.

Reviewer: ```suggestion
modal. This will allow you to choose the repository and its configured default branch.
All branches present when you add the repository will be published.
```

![Start Existing Project][ep]

The name of the project on the network will be fetched from the name of the
repository.

To see if your project was published correctly, you can search for it on the

> [Radicle seed node dashboard][sn]. If it has been correctly replicated by the

Reviewer: ```suggestion
[Radicle seed dashboard][sn]. If it has been correctly replicated by the
```

seed, you be able to see it under the **Projects** section.

> ![Seed Dashboard Search][ss]

Reviewer: This is missing.

Author: It's added in another PR
